packages/metal-dom/src/dom.js

```diff
@@ -254,6 +254,31 @@ class dom {
 	}
 
 	/**
+	 * Verifies if the element is able to trigger the Click event,
+	 * simualting browsers behaviour, avoiding event listeners to be called by dom.triggerEvent method.
+	 * @param {Element} node Element to be checked.
+	 * @param {!String} eventName The event name.
```

Comment:
Primitive types should be lowercase in docs, and don't require a `!` as they're required by default. So this should be just `{string}`. Just FYI, we're following [google's standard](https://developers.google.com/closure/compiler/docs/js-for-compiler) for docs.